scripts/check_style.sh

```diff
@@ -58,6 +63,8 @@ FORMATERROR=$(
     # unqualified move()/forward() checks, i.e. make sure that std::move() and std::forward() are used instead of move() and forward()
     preparedGrep "move\(.+\)" | grep -v "std::move" | grep -E "[^a-z]move"
     preparedGrep "forward\(.+\)" | grep -v "std::forward" | grep -E "[^a-z]forward"
+    # make sure `using namespace std` is not used in INCLUDE_DIRECTORIES
+    grep -nIE -d skip "using namespace std;" ${INCLUDE_DIRECTORIES_JOINED}
```

Comment:
LGTM. Could you fix the missing double quotes as reported by the CI style checker?